pkg/clusterversion/cockroach_versions.go

```diff
@@ -962,6 +966,10 @@ var rawVersionsSingleton = keyedVersions{
 		Key:     V23_2_UseSizedPebblePointTombstones,
 		Version: roachpb.Version{Major: 23, Minor: 1, Internal: 14},
 	},
+	{
+		Key:     V23_2_PebbleFormatVirtualSSTabes,
+		Version: roachpb.Version{Major: 23, Minor: 1, Internal: 16},
+	},
```

Comment:
I think if you add this to the list you have to `./dev gen docs` too

Reply:
Done